### ISSUE TITLE
Enable/Disable mouse capture command not working Windows 10

### ISSUE THREAD

**Issue by TimonPost** [crossterm-rs/crossterm#315]
**Describe the bug**
Mouse events are not enabled on windows.

**To Reproduce**
1. 

```
    execute!(std::io::stdout(), EnableMouseCapture);
    let _r = RawScreen::into_raw_mode().unwrap();

    let mut input = read_sync();

    loop {
        input.next()
    }
```

**Expected behavior**
Mouse even's should be sent with the other events.

**Reason suggestion**
`EnableMouseCapture` and `DisableMouseCapture` commands are executed with `exectute!`. If those are executed a [check ](https://github.com/crossterm-rs/crossterm/blob/master/src/utils/macros.rs#L154) will be done if ANSI is supported and the platform is windows. In the case of windows 10 those are both true and thus is this commands anis code written to the terminal. However, this ansi code is not supported on Windows 10 and the [winapi call](https://github.com/crossterm-rs/crossterm/blob/master/src/input.rs#L443) should be used here instead. 

This bug was introduced with #296 issue.

**Suggested Fix**
Use `TerminalInput::enable_mouse_mode()` instead.

**OS**
 - Windows 10

**Terminal/Shell**
 - Cmd/Powershell

**Comment by TimonPost** [comment]
#326

**Comment by TimonPost** [comment]
Closing fixed in 0.14
